spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/ZipkinAutoConfiguration.java

```diff
@@ -93,6 +94,12 @@ public ZipkinRestTemplateCustomizer zipkinRestTemplateCustomizer(ZipkinPropertie
 		return new DefaultZipkinRestTemplateCustomizer(zipkinProperties);
 	}
 
+	@Bean
+	@ConditionalOnMissingBean
+	InetUtils zipkinInetUtils(){
```

Comment:
I don't think this is needed. Why did you add this?

Reply:
Because you say:
`InetUtils should be a spring bean and be subject to a bean lifecycle (ie: Closable beans will get their close() method called. `